packages/ai-chat-ui/src/browser/chat-input-widget.tsx

```diff
@@ -185,9 +202,11 @@ const ChatInput: React.FunctionComponent<ChatInputProperties> = (props: ChatInpu
         );
 
         editorRef.current = editor;
+        props.setEditorRef(editor);
     };
 
     React.useEffect(() => {
+        props.setEditorRef(undefined);
```

Comment:
This must be moved into the returned function

Reply:
Nice catch, sorry I mismoved it in my previous commit!